app/scripts/views/base.js

```diff
@@ -329,31 +333,67 @@ define(function (require, exports, module) {
     },
 
     /**
-     * Translate a string
+     * Translate a string, output will be HTML escaped.
      *
      * @param {String} text - string to translate
+     * @param {Object} [context] - interpolation context, defaults to
+     * this.getContext();
      * @returns {String}
      */
-    translate (text) {
-      return this.translator.get(text, this.getContext());
+    translate (text, context = this.getContext()) {
+      if (Strings.hasHTML(text)) {
+        const err = AuthErrors.toError('HTML_WILL_BE_ESCAPED');
+        err.string = text;
+        this.logError(err);
```

Comment:
Yup, exactly, the string itself will be part of the message displayed in Sentry.